### PR TITLE
[Php71] Handle variadic check inside function with function_exists() check

### PR DESCRIPTION
**Pull request by samsonasik**
Closes #243 Fixes https://github.com/rectorphp/rector/issues/6516

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/Php71/Rector/FuncCall/RemoveExtraParametersRector.php [hunk 1]
```diff
@@ -143,6 +152,34 @@ private function hasVariadicParameters(array $parameterAcceptors): bool
...
+            });
+
+            return (bool) $this->betterNodeFinder->findFirst($function->stmts, function (Node $node): bool {
+                if (! $node instanceof FuncCall) {
+                    return false;
+                }
+
+                return $this->isNames($node, ['func_get_args', 'func_num_args', 'func_get_arg']);
+            });
+        }
```
Comment (TomasVotruba) [inline rules/Php71/Rector/FuncCall/RemoveExtraParametersRector.php]: :+1: 

Could you extract this to own service? We use similar variadic logic in multiple rules.

Comment (samsonasik) [reply rules/Php71/Rector/FuncCall/RemoveExtraParametersRector.php]: implemented https://github.com/rectorphp/rector-src/pull/244/commits/04897b61123671024e11ed10a9bd83eedf157af9

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**